.github/workflows/pr.yml

```diff
@@ -1,15 +0,0 @@
-name: "Run snippets"
-on:
-  pull_request:
-
-jobs:
-  tests:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-    - uses: cachix/install-nix-action@v14
-      with:
-        nix_path: nixpkgs=channel:nixos-unstable
-        extra_nix_config: |
-          experimental-features = nix-command flakes
-    - run: ./scripts/ci/test.sh
```

Comment:
TODO: we'll need to move this invocation, and potential flakes opt-in, to `ci.nix`.